src/cli-commands/cli-commands.spec.js

```diff
@@ -0,0 +1,26 @@
+
+const mock = require('mock-require');
+const mockCliCommands = require('./cli-commands');
+describe('Entry Plugin', () => {
```

Comment:
Should probably be renamed?

Reply:
🤔 Entry plugins across the board!  lol, #tooMuchHaste